api.md

```diff
@@ -266,7 +330,7 @@ as this is too easy to overuse/use incorrectly.
 2. The user goes to a.com/foo.
```

Comment:
You could add a similar prerendering scenario here, like the following:

1. Site a.com uses a cookie named "SID" to store an auth token for logged-in users.
2. The user goes to a.com/foo while logged in
3. This prerenders a logged-in view of a.com/foo-page-2, which runs the following JS:
4. The user opens a new tab to a.com/bar
5. The user logs out of a.com in that new tab. This causes the prerendered copy of a.com/foo-page-2 held by the first tab to get discarded.
6. The user switches back to their first tab, which is displaying a.com/foo
7. The user clicks a link to a.com/foo-page-2. Because the prerender was discarded, they correctly see a logged-out view of a.com/foo-page-2.